### PR TITLE
feat: added alerts component and styles

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Alerts component.

Simple alerts:
image

Alerts with Icons:
image

On hover (close button - success alert):
image

### CODE REVIEW

File: stylesheets/components/alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    background: var(--simple-alerts-section);
+    padding: 1rem;
+    border-radius: 1rem;
+}
+
+.alerts-simple {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0px;
```
Comment (rahulyadav139) [inline stylesheets/components/alerts/alerts.css:17]: do we need to give gap 0px? isn't it the default setting?

**Comment by rahulyadav139** [comment]
Impressive UI, clean file. Great work.

File: stylesheets/components/alerts/alerts.css (hunk 1 above)
Comment (SinchanaMS) [reply stylesheets/components/alerts/alerts.css:17]: It is. I have a common class in addition that takes 2rem which I'd need for the other components too. Only here I'd need no gap, hence 0px.

File: stylesheets/components/alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    justify-content: space-between;
+    align-items: center;
+    padding: auto 1rem;
+}
+
+.icon-text{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: 10px;
```
Comment (aiman-ansari) [inline stylesheets/components/alerts/alerts.css:79]: Use rem instead of px

**Review by aiman-ansari — APPROVED** [review]
Everything is good code is very clean code just remove extra spaces .

File: stylesheets/components/alerts/alerts.css (hunk 2 above)
Comment (SinchanaMS) [reply stylesheets/components/alerts/alerts.css:79]: Thank you, Aiman! 
It was recommended to use px if the value is <=10px, hence keeping the same.

**Review by m26unkwn — APPROVED** [review]
Clean UI good to go

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

**Pull request merged**